api/rest-api/README.md

```diff
@@ -95,7 +95,6 @@ When calling a production Rocket.Chat server, ensure it is running via HTTPS and
 | `/api/v1/channels.addOwner` | Gives the role of owner for a user in the current channel. | [Link](methods/channels/addowner.md) |
 | `/api/v1/channels.anonymousread` | Gets the messages in public channels to an anonymous user | [Link](methods/channels/anonymousread.md) |
 | `/api/v1/channels.archive` | Archives a channel. | [Link](methods/channels/archive.md) |
-| `/api/v1/channels.cleanHistory` | Cleans up a channel's history, requires special permission. | [Link](https://github.com/RocketChat/docs/tree/4f704d5da4a2d5bcfe2cd2d2591edd12f5d41cd6/developer-guides/rest-api/channels/cleanhistory/README.md) |
```

Comment:
Not sure if this was moved, or never existed, but it looks like this is now in `rooms.cleanHistory`